### PR TITLE
Updated LSTM Cell inputs description to 3D tensor

### PR DESCRIPTION
**Pull request by Chirag-Shilwant**
Actually the LSTM Cell takes a 3D tensor as input as we can see on line number 801 but the description at line number 860 mentions that the input should be a 2D tensor which is incorrect. The LSTM cell gives error with 2D tensor input. Hence modified the description as 3D tensor.

### CODE REVIEW

**Comment by Chirag-Shilwant** [comment]
@googlebot  I signed it!

**Comment by qlzh727** [comment]
No, this is not correct. LSTMCell only take 2D input and it will only process one timestep (so there isn't a dim for timestep in the input). On the otherhand, LSTM layer will take 3D tensor, and the current docstring is correctly reflecting that.

**Comment by Chirag-Shilwant** [comment]
@qlzh727 But the example used on the tensorflow official docs itself uses 3D tensor as input for LSTMCell.
[https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTMCell](url)

**Comment by qlzh727** [comment]
> @qlzh727 But the example used on the tensorflow official docs itself uses 3D tensor as input for LSTMCell.
> [https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTMCell](url)

Please pay attention to the sample code, the LSTMCell is wrapped in a RNN layer, which takes 3D tensor as input. When the input reaches the cell, the input shape is 2D. You can debug the code step by step to see the shape of input.
